### ISSUE TITLE
Badges in README

### ISSUE THREAD

**Issue by drkennetz** [drkennetz/go-slack-chat-gpt3#5]

**Comment by drkennetz** [comment]
@hemantsoni42 absolutely, thanks for offering! Do you need anything from me to get started?

**Comment by hemantsoni42** [comment]
yes can you please provide me some information and links.
Like if you have a sponsors, backers, build, service testing, coverage, social or chat link like slack, glitter, discord and social media link like twitter, linkedin etc

**Comment by hemantsoni42** [comment]
@drkennetz I already submitted a pull request.
Please let me know if we want anything change in updated readme

**Comment by hemantsoni42** [comment]
Yeah Sure I am always open to contribute
